### ISSUE TITLE
cannot be resolved to an item or type

### ISSUE THREAD

**Issue by agh251256** [openhab/openhab-core#1154]
using Openhab 2.5 Build #1728 and earlier, Raspberry PI, Kernel Debian Stretch 4.19

After every update I've the Issue, that every unbound Item will not register to OpenHab. I'm getting following Error in the openhab.log

 `2019-09-29 15:01:37.147 [ERROR] [ntime.internal.engine.RuleEngineImpl] - Rule 'alle_Rolladen_Automatik AUS': The name 'All_Roll_ModeAutoTmp' cannot be resolved to an item or type; `

Thats is the part of the Item-File:
`Switch All_Roll_ModeAutoTmp`

After reloading the Item-File, e.g. by editing it, all errors will gone and the rules works.

**Comment by lolodomo** [comment]
IMHO it is just the usual issue due to random oder at startup, you rule was probably loaded before the item.

**Comment by agh251256** [comment]
It’s only after an update and starting up the system. After reloading the item file after editing it, it works fine

 

Von: lolodomo <notifications@github.com> 
Gesendet: Sonntag, 20. Oktober 2019 16:51
An: openhab/openhab-core <openhab-core@noreply.github.com>
Cc: agh251256 <a.guhr@aghnet.de>; Author <author@noreply.github.com>
Betreff: Re: [openhab/openhab-core] cannot be resolved to an item or type (#1154)

 

IMHO it is just the usual issue due to random oder at startup, you rule was probably loaded before the item.

**Comment by openhab-5iver** [comment]
I believe this is a duplicate of or related to #865.

**Comment by agh251256** [comment]
Maybe, I can't discuss this, I'm only use openhab and see the errors. I don't clear the cache before updating.

**Comment by agh251256** [comment]
the bug seems to be fixed. No longer occurs, only the CPU average is extremly.
